src/output.jl

```diff
@@ -9,17 +9,19 @@ function plot_abundances(SD::Simulation_Data)
     carry = fill(
         sum(SD.species[1].vars.carry .* SD.species[1].vars.habitat), length(total_abundance)
     )
-    f = Figure(; fontsize = 24)
-    ax = Axis(f[1, 1], title="Species Abundance over Time", 
-    xlabel = "timestep",
-    ylabel = "total abundance")
-
-    lines!(ax, x, total_abundance, label = "Abundance", linewidth = 4)
-    lines!(ax, x, carry, label = "Carrying Capacity", linewidth = 4)
-    CairoMakie.ylims!(ax, 0, maximum(total_abundance)+20)
+    f = Figure(; fontsize=24)
+    ax = Axis(
+        f[1, 1];
+        title="Species Abundance over Time",
+        xlabel="timestep",
+        ylabel="total abundance",
+    )
+
+    lines!(ax, x, total_abundance; label="Abundance", linewidth=4)
```

Comment:
Found another bug here, because it doesn't handle `missing`

```
TypeError: non-boolean (Missing) used in boolean context

    ylims!(::Makie.Axis, ::Tuple{Int64, Missing})@axis.jl:1255
    ylims!(::Makie.Axis, ::Int64, ::Missing)@axis.jl:1270
    plot_abundances(::MetaRange.Simulation_Data)@output.jl:22
    top-level scope@[Local: 1](http://localhost:1234/edit?id=2f728bd4-733b-11ee-205e-6745835035ab#)[inlined]
```

Reply:
(this was in the bavaria example)

Reply:
I'll make this its own issue so we can go ahead and merge this branch regardless